package.json

```diff
@@ -10,15 +9,20 @@
     "datatables.net-rowreorder-bs4": "^1.2.8",
     "jquery": "^3.5.0",
```

Comment:
What do we still use jquery for? Is there legacy javascript code that is using it?

Reply:
Yep, some legacy stuff, most of which is just the selectors, so fairly easy to update. 
However datatables also has a dependency on jquery, as does the version of bootstrap we are using.When it comes to updating bootstrap I know the newer versions don't support the latest versions of FF and Chrome to support XP, but I don't know if that is 'catastrophically bad rendering' or 'just doesn't look quite as nice'.